### ISSUE TITLE
about train/test result

### ISSUE THREAD

**Issue by Adorablepet** [daizuozhuo/batch-feature-erasing-network#12]
Thank you very much for sharing the code. I have been training less than 95.3% when I use your script to do training. I can only train about 94.5%. The environment I use is pytorch1.0.1, 4 v100, can you help me analysis? Thanks!

**Comment by Adorablepet** [comment]
I modified your learning rate strategy and now reach around ~95%.

**Comment by wzjiang** [comment]
Hello. I have a similar problem. How did you change it?

**Comment by sksksk1748** [comment]
Hello, Validation dataset are need ?
